config-example.json

```diff
@@ -15,6 +15,7 @@
     "voting_delay_min": "2d",
     "voting_delay_max": "10d",
     "staging_checks": 2,
+    "approval_url": "http://example.com",
```

Comment:
Please use an empty string (disabling the feature) rather than wrong URL by default. Please do not forget to update README as well.